src/content/docs/new-relic-solutions/new-relic-one/install-configure/uninstall-agent.mdx

```diff
@@ -24,7 +24,7 @@ If you encounter problems, see [support.newrelic.com](https://support.newrelic.c
 ## Before you uninstall [#downgrade]
 
 <Callout variant="important">
-  For paying New Relic accounts, note that an alternative option to uninstalling is to [downgrade your account](/docs/accounts/accounts-billing/account-setup/downgradecancel-account) to a less expensive or free tier.
+  For paying New Relic organizations, an alternative to uninstalling is to [downgrade your account](/docs/accounts/accounts-billing/account-setup/downgradecancel-account) to a less expensive edition.
```

Comment:
So, you can't downgrade to a free account anymore?

Reply:
i guess i could make that more clear. i thought 'less expensive' included free but I can clarify that more